lib/pure/algorithm.nim

```diff
@@ -208,7 +242,10 @@ proc lowerBound*[T, K](a: openArray[T], key: K, cmp: proc(x: T, k: K): int {.clo
   ## * `upperBound proc<#upperBound,openArray[T],K,>`_
```

Comment:
Wrong link.

It should be:
```
   ## * `upperBound proc<#upperBound,openArray[T],T>`_
```

The same goes for a link to `lowerBound` on the line 289.

Reply:
56c897f968f9215006521e10f310bd03d09901fe